.github/workflows/platform_tests.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        go-version: [1.14, 1.19]
+        go-version: [1.14, 1.17]
```

Comment:
Can't we have 1.14, 1.17 and 1.19? As the gopherjs test is only enable for 1.17, that would work, no?

Reply:
That is adding a lot of time to the compiling.
Really I think we should use 19 for most, and add another test run for the web build which can use 1.17

Reply:
That make sense indeed.

Reply:
We had more than two Go versions earlier but it gets very slow in the long run. I prefer to keep it as it is

Reply:
Yeah, I agree with @andydotxyz. That seems like a good idea. Will add that to my list

Reply:
You are also free to tackle that @Bluebugs if you are interested :)